CMakeLists.txt

```diff
@@ -310,6 +309,7 @@ if (CMAKE_BUILD_TYPE STREQUAL "Debug" AND NOT NOVA_IN_SUBMODULE)
     set(NOVA_TEST ON)
 endif()
 
+set(NOVA_TEST ON)
```

Comment:
Thats not good, add at least a check if nova is in a submodule.

Reply:
We need to talk about this actually. The way you were setting NOVA_TEST doesn't work at all for msvc. Is there an easy way to support msvc or do I have to pass the variable on the command line?

Reply:
I'm passing the variable to the cmake command line, better said Clion does it for me.